HealthcareSystem/UserService/Model/DoctorAccount.cs

```diff
@@ -13,7 +14,7 @@ protected override void Validate()
             base.Validate();
             if (DateOfEmployment.CompareTo(DateTime.Now) > 0)
             {
-                throw new Exception("Date of employment not valid.");
+                throw new ValidationException("Date of employment not valid.");
             }
```

Comment:
What did we decide in the end, are we checking dates or avoiding it because of the DateTime.Now dependency?